pkg/strace/internal/abi/abi_linux.go

```diff
@@ -19,7 +19,7 @@ import (
 	"syscall"
```

Comment:
why internal? the abi package struck me as something that could be generally useful.